internal/registry.go

```diff
@@ -9,6 +9,10 @@ import (
 type internalSender interface {
 	// Sends a single metric to Wavefront with optional timestamp and tags.
 	SendMetric(name string, value float64, ts int64, source string, tags map[string]string) error
+
+	// Sends a delta counter (counter aggregated at the Wavefront service) to Wavefront.
+	// the timestamp for a delta counter is assigned at the server side.
+	SendDeltaCounter(name string, value float64, ts int64, source string, tags map[string]string) error
```

Comment:
Same question here as well. Can omit `ts` since as the comment suggests, the timestamp for a delta counter is assigned at the server side.